be/src/olap/tablet_manager.cpp

```diff
@@ -562,13 +562,22 @@ OLAPStatus TabletManager::_drop_tablet_unlocked(TTabletId tablet_id, SchemaHash
 OLAPStatus TabletManager::drop_tablets_on_error_root_path(
         const std::vector<TabletInfo>& tablet_info_vec) {
     OLAPStatus res = OLAP_SUCCESS;
-    for (int32 i = 0; i < _tablets_shards_size; i++) {
+    if (tablet_info_vec.empty()) { // This is a high probability event
+        LOG(INFO) << "tablet_info_vec is empty, return directly.";
```

Comment:
I think this log can be removed?

Reply:
ok， i will test this pr in our cluster. And then remove this log.